docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
If the file is authored in VI, you should use one space after a period. Otherwise two spaces a permissible. This clearly proves that VI is the superior editor. ;-)